andela/suyabay
# #121836665 Fix social network not working on some env and notification issue

#### What does this PR do?
- Fix Social network authentication not working in some environment.
- Allow users to receive notification when they log in.
#### Description of Task to be completed?
- Modify the route to add a callback route.
- Modify the `OuthController` to remove the unnecessary piece of code.
  -Modify the `SignUpLogin.js` to add a swal alert in order for the user to receive notification when they successfully login.
#### How should this be manually tested?
- Go to `http://suyabay-staging.herokuapp.com/` then click on `login` link. This will take you to the login page. Sign in traditionally and you will see a notification- `You have successfully Sign In` redirecting the authenticated user to the home page.
- For the Social network authentication now working. In the Login page, Sign in by using the social network (either facebook or twitter). This will Sign in the user with notification depending on whether the user is a returning user or a new user. For a returning user, notification seen is - `You have successfully Sign In`, for a new user, notification seen is - `You have successfully Sign Up`.
#### Any background context you want to provide?

none
#### What are the relevant pivotal tracker stories?
#121836665
#### Screenshots (if appropriate)

![screen shot 2016-06-20 at 6 34 51 pm](https://cloud.githubusercontent.com/assets/14159420/16204723/24eca13e-3718-11e6-97ab-ef13fc51bbeb.png)
#### Questions:

none

[@unicodeveloper](https://github.com/unicodeveloper)
[@andela-oadebayo](https://github.com/andela-oadebayo)
